src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -730,5 +731,253 @@ unsigned int ThermalCluster::precision() const
 {
     return 0;
 }
+
+bool ThermalCluster::loadFromProperty(ThermalCluster& cluster, const IniFile::Property* p)
```

Comment:
Please introduce a `std::map<std::string, function_ptr>` or something similar.

functionMap[p->property](p->value)

This switch..case is horrible to maintain.